components/movementsensor/imuvectornav/imu.go

```diff
@@ -191,34 +192,29 @@ func newVectorNav(
 	if err != nil {
 		return nil, errors.Wrap(err, "couldn't configure deltaV register")
 	}
+
 	// tare the heading
 	err = v.vectornavTareSPI(ctx)
 	if err != nil {
 		return nil, err
 	}
-	smp := uint(100)
-	if v.polling > 0 {
-		smp = uint(v.polling)
-	}
-
 	// compensate for acceleration bias due to misalignement
-	err = v.compensateAccelBias(ctx, smp)
+	err = v.compensateAccelBias(ctx, v.polling)
 	if err != nil {
 		return nil, err
 	}
 	// compensate for constant DV bias in mesurament
-	err = v.compensateDVBias(ctx, smp)
+	err = v.compensateDVBias(ctx, v.polling)
 	if err != nil {
 		return nil, err
 	}
 
-	// optionally start a polling goroutine
-	if pfreq > 0 {
-		logger.CDebugf(ctx, "vecnav: will pool at %d Hz", pfreq)
+	if pollFreq > 0 {
+		logger.CDebugf(ctx, "vecnav: will poll at %d Hz", pollFreq)
 		waitCh := make(chan struct{})
-		s := 1.0 / float64(pfreq)
+		pollHertz := 1.0 / float64(pollFreq)
```

Comment:
This is actually in units of seconds since it's 1/frequency

Reply:
will rename to `pollPerSecond`